tests/corrupted_stamp_test.cpp

```diff
@@ -239,13 +252,14 @@ void test_read_corrupted_stamp(ioremap::elliptics::newapi::session &s, const dne
 
 		return test_write_result{std::move(k),
 		                         check_skipped ? 0 : ec,
-		                         constants::HEADERS_SIZE + to_add - to_remove,
+		                         get_headers_size(test_case.dc_flags),
+		                         get_headers_size(test_case.dc_flags) + to_add - to_remove,
 		                         std::move(async)};
 	};
 
 	std::vector<test_write_result> write_results;
 
-	size_t id = 0;
+	int id = 0;
```

Comment:
Why is it `int` now?

Reply:
Thanks, I'll revert.
I reverted some changes and occasionally pasted it.